src/main/dml/fenixedu-academic.dml

```diff
@@ -4635,6 +4635,11 @@ public class log.StudentRegistrationTransferLog {
     protected String who;
 }
 
+
+class candidacy.FirstTimeDocumentsConfiguration {
+	JsonElement configurationProperties;	
```

Comment:
Be careful here, as you should **NEVER** expose `JsonElement`s directly, as they are not immutable and **will** cause issues.